### DISCUSSION TITLE
Is it possible to preview a revealjs presentation forcing no-browser and setting port?

### DISCUSSION THREAD

**Discussion by aborruso** [quarto-dev/quarto-cli#3109] [Q&A] [answered]
Hi,
if I run `quarto preview --no-browser --port 6074 ./file.qmd` I have `file is not a website or book project`.

How to use `--no-browser --port` for a revealjs presentation?

Thank you

**Comment by mcanouil** [comment, marked as answer]
Try with `quarto preview ./file.qmd --no-browser --port 6074`.
Somehow the order matters, options are after the file.

**Reply by aborruso** [reply]
thank you
